### PR TITLE
Add searchbox funtionality to filter logs

### LINKED ISSUE #341: Filter Deployment logs
**Is your feature request related to a problem? Please describe.**
Cyclops shows logs for running pods [here](https://github.com/cyclops-ui/cyclops/blob/11185d7871fc0957ffab78bfd5f54d783fbb5fc3/cyclops-ui/src/components/k8s-resources/Deployment.tsx#L114). Scope of the task is to add a search for logs that you can use to filter out the log you are looking for.

**Describe the solution you'd like**
Add a search bar to the top of the logs modal and show all logs that contain the search input.

### PR DESCRIPTION
**Pull request by Sheikh-Abubaker**
closes #341 

## 📑 Description

This PR adds a searchbox for logs that you can use to filter out the log you are looking for.

## ✅ Checks
- [x] I have tested my code (provide screenshots or screen recordings of a working solution)
- [x] I have performed a self-review of my code

### CODE REVIEW

**Comment by Sheikh-Abubaker** [comment]
@quest-bot loot #341

File: cyclops-ui/src/components/k8s-resources/common/PodTable/PodLogs.tsx [hunk 1]
```diff
@@ -71,6 +72,15 @@ const PodLogs = ({ pod }: PodLogsProps) => {
...
                 readOnly={true}
+                commands={[
+                  {
+                    name: "showSearchBox",
+                    bindKey: { win: "Ctrl-F", mac: "Command-F" },
+                    exec: function (editor) {
+                      editor.execCommand("showSearchBox");
+                    },
+                  },
+                ]}
```
Comment (petar-cvit) [inline cyclops-ui/src/components/k8s-resources/common/PodTable/PodLogs.tsx:83]: Can we remove this? I think importing ace-builds searchbox does the job and binds it to `Ctrl-f|Command-f` -> https://github.com/ajaxorg/ace-builds/blob/master/src/ext-searchbox.js

Comment (Sheikh-Abubaker) [reply cyclops-ui/src/components/k8s-resources/common/PodTable/PodLogs.tsx]: That's right!! it does, updated the code checkout the latest commit.

**Review by petar-cvit — APPROVED** [review]
Thanks @Sheikh-Abubaker 🧡

**Comment by Sheikh-Abubaker** [comment]
Hey @petar-cvit this issue had a quest label right? so how does it work, it was mentioned that the  contributor would be rewarded🤔

**Pull request merged**